gabela/core/Session.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Gabela\Core;
+
+class Session
+{
+    public function getCurrentUserId()
+    {
+        return $_SESSION['user_id'];
+    }
+
+    public function getCurrentUsername()
+    {
+        return $_SESSION['user_name'];
+    }
+
+    public function getCurremtUser()
```

Comment:
Function name needs to be renamed from **getCurremtUser** to **getCurrentUser**

Reply:
Changed